rest/api_test.go

```diff
@@ -3884,10 +3885,11 @@ func TestNumAccessErrors(t *testing.T) {
 	assert.Equal(t, float64(1), numAccessErrors)
 }
 
-var prt RestTester
-
 func Benchmark_RestApiGetDocPerformance(b *testing.B) {
 
+	var prt RestTester
+	prt.Close()
```

Comment:
Is this missing a defer?

Reply:
Yes. Not sure why it was passing without though?

Reply:
Benchmarks only get run when using `go test -bench=.`